### PR TITLE
Apply waf rules

### PR DESCRIPTION
**Pull request by VolkerBohne**
Starts with a basic set of rules attached to CDN distribution

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -33,10 +33,18 @@ module "lambdaedge_function" {
   config = var.environment
 }
 
+module "waf_rules" {
+  source = "./modules/waf_rules"
+  providers = {
+    aws = aws.us
```
Comment (michaelkrieg) [inline terraform/main.tf]: aws.us should be more detailed imho.
just in case you'd wanted to address the exact region-(code) as well

**Review by michaelkrieg — COMMENTED** [review]
in summary, this looks good.

File: terraform/main.tf (hunk 1 above)
Comment (VolkerBohne) [reply terraform/main.tf]: fixed.

**Review by michaelkrieg — APPROVED** [review]
yeah thats the way it goes 🎸

**Pull request merged**